src/@next/Skeleton/components/SkeletonText/SkeletonText.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { Variant as TypographyVariant } from '../../../Typography';
+import { SkeletonTextContainer, StyledSkeletonText } from './SkeletonTextStyle';
+
+export interface SkeletonTextProps {
+  rows?: number;
+  width?: string | string[];
+  variant?: TypographyVariant | TypographyVariant[];
+}
+
+export const SkeletonText = ({
+  rows = 4,
+  width = '100%',
+  variant = 'body1',
+}: SkeletonTextProps) => {
+  const widthArray =
+    typeof width === 'string'
+      ? Array.from({ length: rows }, () => width)
+      : width;
+
+  const variantArray: TypographyVariant[] =
+    typeof variant === 'string'
+      ? Array.from({ length: rows }, () => variant)
+      : variant;
+
+  const renderTexts = () => {
+    return [...Array(rows)].map((_, index) => {
+      return (
+        <StyledSkeletonText
+          key={`skeleton-text-${index}`}
+          width={widthArray[index]}
+          variant={variantArray[index]}
```

Comment:
what will happen if the no of rows != label variant?

Reply:
it won't render the row. we can ask for a default for this one as well? for the width, it defaults to 100%.